library/Vanilla/ReCaptchaVerification.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license Proprietary
+ */
+
+namespace Vanilla;
+
+use Garden\Http\HttpClient;
+
+/**
+ * Class reCaptchaVerification
+ */
+class ReCaptchaVerification {
+
+    const RECAPTCHA_V3_URL = "https://www.google.com/recaptcha/api/siteverify";
+
+    /** @var HttpClient */
+    private $httpClient;
+
+    /**
+     * ReCaptchaVerification constructor.
+     */
+    public function __construct() {
+        $this->httpClient = new HttpClient();
```

Comment:
We should be dependency injecting this (so any global middlewares get applied.)

Reply:
done